src/main/resources/META-INF/resources/primefaces/datepicker/1-datepicker.js

```diff
@@ -144,6 +144,17 @@ PrimeFaces.widget.DatePicker = PrimeFaces.widget.BaseWidget.extend({
         this.input.data(PrimeFaces.CLIENT_ID_DATA, this.id);
     },
 
+    updateDisabledDates: function(disabledDates) {
```

Comment:
.. or should we simply call this `setDisabledDates`?

Reply:
I like setDisabledDates, can you also create a duplicate method called `setDisabledDays` that I think takes an array like [0,1] to turn off all Saturday's and Sundays if you look in the showcase and then refactor if they can share code?

Reply:
Also don't forget to JSDoc the method since its public and poeple will want to know what `disabledDates` parameter is.